DigitalLearningSolutions.Data.Migrations/202206281412_AddForeignKeysToSupervisorDelegatesAndCandidateAssessments.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202206281412)]
+    public class AddForeignKeysToSupervisorDelegatesAndCandidateAssessments : Migration
+    {
+        public override void Up()
+        {
+            Alter.Table("SupervisorDelegates")
+                .AddColumn("DelegateUserID").AsInt32().Nullable()
+                .ForeignKey("Users", "ID");
+
+            Alter.Table("CandidateAssessments")
+                .AddColumn("DelegateUserID").AsInt32().Nullable()
+                .ForeignKey("Users", "ID")
+                .AddColumn("CentreID").AsInt32().Nullable()
+                .ForeignKey("Centres", "CentreID");
+
+            Execute.Script(
+                "DigitalLearningSolutions.Data.Migrations/Scripts/HEEDLS-932-PopulateNewKeysOnSupervisorDelegatesAndCandidateAssessments.sql"

```

Comment:
I could not get the migration to run, because this relative path resolved to the wrong absolute path: trying to start the localhost website resulted in the path being `/path/to/my/repo/DigitalLearningSolutions.Data.Web/DigitalLearningSolutions.Data.Migrations/Scripts/HEEDLS-932-PopulateNewKeysOnSupervisorDelegatesAndCandidateAssessments.sql`

You can fix this by changing `Execute.Script` to `Execute.Sql`, putting the SQL from `HEEDLS-932-PopulateNewKeysOnSupervisorDelegatesAndCandidateAssessments.sql` in as an argument, and removing that file